scripts/monkey_all.py

```diff
@@ -0,0 +1,9 @@
+import inspect
```

Comment:
Can you explain what does this script do?

Reply:
MonkeyType takes in information to determine accurate typings. I have that script to run every test, class, and function (Without args as I can't simulate those) to try and give it as much info as possible. So I MonkeyType run that function.

Reply:
Add comment on what does this do here.

Reply:
On it.